redfish-core/lib/sensors.hpp

```diff
@@ -52,6 +53,9 @@ static constexpr std::string_view sensors = "Sensors";
 static constexpr std::string_view thermal = "Thermal";
 } // namespace node
 
+// Regular expression that matches power supply input power sensor names
+static const std::regex powerSupplyInputPowerPattern{"ps[0-9]+_input_power"};
```

Comment:
There has been some patches upstream in bmcweb to move away from regexes. e.g. https://gerrit.openbmc.org/c/openbmc/bmcweb/+/51094
Since this is a temporary patch.. Don't think I will enforce but just an FYI

Reply:
OK thanks for the information.

In the referenced review, they replaced a very complex regex for URLs with a built-in library, which makes a ton of sense.  

Are they removing regex's in general though?  That seems strange.  In a case like the sensor name pattern above, a regex provides a very simple and powerful solution.  The equivalent std::string parsing solution would be longer and more error prone IMO.  But I'll abide by it for upstream code if that is what the community wants.

Reply:
I am fine with this for now, since this whole regex is going away for a proper D-Bus <-> Redfish